projects/packages/forms/src/contact-form/css/grunion.css

```diff
@@ -692,8 +733,8 @@ on production builds, the attributes are being reordered, causing side-effects
 }
 
 /* Fix padding for new forms in animated style */
-.contact-form .is-style-animated .grunion-field-wrap:not(.no-label) > .wp-block-jetpack-input,
-.contact-form .is-style-animated .grunion-field-wrap:not(.no-label) .wp-block-jetpack-input,
+.contact-form .is-style-animated .grunion-field-wrap:not(.no-label) > .wp-block-jetpack-input:not(.select),
+.contact-form .is-style-animated .grunion-field-wrap:not(.no-label) .wp-block-jetpack-input:not(.select),
```

Comment:
Does this do anything? (This is specifying `.select` not a `select` element)

Reply:
Yes, it targets all inputs except the select wrapper, which is the element that gets all the styles. Here is what it looks like:

```html
<div class="contact-form__select-wrapper select wp-block-jetpack-input has-text-color has-background has-border-color has-luminous-vivid-amber-border-color grunion-field has-placeholder" style="color:#00d084;background-color:#8ed1fc; font-size:clamp(1.119rem, 1.119rem + ((1vw - 0.2rem) * 1.485), 1.75rem); border-radius:6px;border-style:solid;border-width:11px;">
</div>
```

It could easily be replaced with `.contact-form__select-wrapper`.

Reply:
> It could easily be replaced with .contact-form__select-wrapper.

I did that